package.json

```diff
@@ -219,8 +227,8 @@
     "react-floki": "^1.0.70",
     "rimraf": "2.6.3",
     "shelljs": "0.8.3",
-    "storybook-addon-intl": "^2.4.1",
-    "storybook-addon-smart-knobs": "^5.0.0",
```

Comment:
This is not required now ? Why is this removed ?

Reply:
@storybook/addon-controls is required to be added or it comes by default ?

Reply:
@storybook/addon-controls needs to be added ( recommended )